tests/test_matrices_space.py

```diff
@@ -59,6 +59,15 @@ def test_commutator(self):
         result = self.space.commutator(x, [x, y, z])
         expected = gs.array([gs.zeros((3,3)), z, -y])
         self.assertAllClose(result, expected)
+    
+    @geomstats.tests.np_only
+    def test_transpose(self):
+        tr = self.space.transpose
```

Comment:
let's not rename functions: it might confuse other developers who are used to see transpose and array

Reply:
I actually think it's more readable:
+ it's only renamed inside the body function, for the next couple lines, and not globally
+ the following lines would actually be unreadable with the full names
+ this is exactly like declaring a local variable `tr_a = MatricesSpace.transpose(a)`, except that by letting `tr = MatricesSpace.transpose` I can reuse `tr` and apply it to `b`

Reply:
@johmathe any opinion on this renaming?

Reply:
This violates the style guide originally agreed and can lead to very bad things because of confusions between variable names and function names. Please revert this PR ASAP. As mentioned in the google style guide the most important thing is consistency across the code base.